kiliautoml/utils/helpers.py

```diff
@@ -334,7 +335,26 @@ def get_keys(my_dict):
                 table_int.append("nan")
         table.append(table_int)
         table_int = []
-    print(tabulate(table, headers=[job_name] + keys))
+    headers = [job_name] + keys
+    return headers, table
+
+
+def print_and_save_evaluation(
+    job_name: JobNameT, evaluation: EvalResultsT, results_folder: Optional[str] = None
+):
+    headers, table = create_table(job_name, evaluation)
+    print(tabulate(table, headers=headers))
+
+    if results_folder is not None:
```

Comment:
else ?

Reply:
We already printed the evaluation table so there is no need for anything else here.